ci_scripts/check_documentation.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+log_file="${TMPDIR}/jazzy_status.log"
+
+if ! command -v jazzy > /dev/null; then
+  if [[ "${CI}" != "true" ]]; then
+    echo "ERROR: Please install jazzy:"
+    echo "https://github.com/realm/jazzy"
+    exit 1
+  fi
+
+  echo "Installing jazzy..."
+
+  gem install jazzy || die "Executing \`gem install jazzy\` failed"
```

Comment:
need to copy the `die` function (probably bring in the `info` function too). see `export_builds.sh`.

additionally, you can replace all of the `echo "ERROR: ..." ; exit 1` with calls to `die`